### ISSUE TITLE
Tools are too cheap to buy in other colonies

### ISSUE THREAD

**Issue by victorxup** [We-the-People-civ4col-mod/Mod#905]
To be discussed:

When you buy Tools in other colonies, their price is waaay cheaper than in Europe.
While this could have been designed intentionally, I suppose player should not have such a source of cheap tools.

**Comment by Schmiddie1979** [comment]
What do you mean with "buying in other colonies"? You mean Port Royal and Africa or what?

**Comment by victorxup** [comment]
> What do you mean with "buying in other colonies"? You mean Port Royal and Africa or what?

I mean colonies of AI players. Easily reproducible at higher difficulties

**Comment by raystuttgart** [comment]
Yeah, that is known and actually the main reason why trading wiht AI cities is even useful.
But it makes sense honestly. Because if they overproduce, they want to get rid of it.

Also it will improve the political relations with the AI, so I kind of like it and see little problem for gameplay.
Also it will be limited to the amounts they can produce ... it is not unlimited like in Europe.

Also of course it is cheaper, because if they would have to ship it to Europe they would have transportation costs.
Also they would have to sell there for the sell prices and not the buy prices ... 

When they would sell huge amounts of those in Europe, the prices there would also drop.
So why not sell some of it to the colonial neighbours?

**Summary:**
I do not really see a big issue in it. (It is limited and does not hurt AI ... as it may overproduce it anyways.)
Quite the contrary, I like it ... as it gives a gameplay incentive to trade peacefully.

**In other words:**
I like it the way it is and consider it good for gameplay.
It gives me an incentive to stay peaceful with other Europeans and trade with them.

**Comment by raystuttgart** [comment]
Can we close this issue?
If not, please explain how this causes an issue for gameplay.
Also no player is forced to trade with other colonies at all if he does not like it.

**Comment by victorxup** [comment]
Most of that makes sense.  And indeed the issue is quite minor.

The part I want to highlight is during early stages of the game: 
When AI **brings Tools from Europe** (where it was bought for even lesser price), **not produces**.
So Tools can be bought for prices, which **were meant for AI, not for Human Player**. A bit of exploit IMHO, especially for early game, when even low quantities can give some advantage.

In my opinion, price could be rebalanced to be a bit higher - just to be comparable with european. 
Tools in AI colonies are still closer to the player than Europe - so trade doesn't lose sense.

But anyway, as I mentioned, this is not a high priority issue.
(Not sure about it, maybe such low prices for AI are Handicap related, thus scaled with difficulty?)

**Comment by victorxup** [comment]
Of course the issue can be closed in case there was already consensus in the Team about it

**Comment by raystuttgart** [comment]
Well AI will never differentiate in its pricing considering "brought from Europe" or "produced in Colonies".
But I highly doubt that anybody thinks this is worth time and effort to work on it ...

We never had any complaint so far in all those years ... and the logic had never changed.
Most people actually do not even trade Yields with other Colonies at all as it seems.

**Comment by victorxup** [comment]
As I already mentioned - it makes sense.
And it is not even close a complaint))

**Comment by raystuttgart** [comment]
Ok, then let us close it please.

1) We have very vew active programmers around anyways.
2) We are trying to publish a release and will not waste efforts for things that are not really an issue

Thanks for your understanding.
